subscription-manager.spec

```diff
@@ -681,6 +681,8 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
     %tmpfiles_create %{_tmpfilesdir}/subscription-manager.conf
 %else
     %systemd_post rhsmcertd.service
+    %systemd_post rhsm.service
+    systemctl restart rhsm.service
```

Comment:
Why do you call `systemctl restart rhsm.service`? Why the macro `%systemd_post rhsm.service` is not enough?